### ISSUE TITLE
Morphology benchmarks to torch (CPU), for example kornia

### ISSUE THREAD

**Issue by ogencoglu** [neuro-ml/imops#44]
kornia has [torch-native morphology ops](https://kornia.readthedocs.io/en/latest/morphology.html). Benchmarks to that would be great. I guess accounting numpy to torch and back to numpy conversions to have apples to apples. All on CPU obviously.

**Comment by vovaf709** [comment]
Yes, good idea, thanks! We will benchmark against kornia and some other popular libraries very soon #33

**Comment by ogencoglu** [comment]
Maybe also for openCV

**Comment by ogencoglu** [comment]
Also possibly with [mahotas](https://github.com/luispedro/mahotas), [smil](https://smil.cmm.minesparis.psl.eu/doc/p480.html), [pyvips](https://github.com/libvips/libvips/wiki/Speed-and-memory-use)  which all claim to be really fast regarding these kind of operations.

**Comment by vovaf709** [comment]
Thx, nice suggestions! We will include them in benchmark list
